README.md

```diff
@@ -1 +1,58 @@
-# go-appr
+## go-appr
+`go-swagger` generated client bindings for `App Registry`. This package can be used to talk to `quay.io` or [appr](https://github.com/app-registry/appr) to pull down application packages.
+
+Swagger spec file has been obtained from here -  [appr-api-swagger.yaml](https://github.com/app-registry/appr/blob/master/Documentation/server/appr-api-swagger.yaml)
+
+### Prerequisite
+* [go-swagger](https://github.com/go-swagger/go-swagger)
+
+```bash
+# Install go-swagger
+go get -u github.com/go-swagger/go-swagger/cmd/swagger
+```
+
+### Generate Client Bindings
+```bash
+# change directory to the root of the go-appr repo.
+swagger generate client --spec=./appr.spec.yaml --name=appregistry --api-package=appr --client-package=appregistry
+```
+
+### Example
+To run the example below, you would need an `appr` server up and running. Visit https://github.com/app-registry/appr for more information on how to run an app registry server on local environment.
```

Comment:
To follow up on the comment above, having an appr server up and running would be a prerequisite for using go-appr